docs/reference/execute-methods.md

```diff
@@ -199,11 +199,65 @@ bundleId | string | yes | The bundle identifier of the application to be removed
 
 #### Returned Result
 
-Either `true` if the app was successfully uninstalled, otherwise `false`
+Either `true` if the application was successfully uninstalled, otherwise `false`
 
 ### mobile: launchApp
 
-Executes the given app on the device under test. If the app is already running then it would be activated. If the app is not installed or cannot be launched then an exception is thrown.
+Executes the given application on the device under test. If the application is already running then it would be activated.
+If the application is not installed or cannot be launched then an exception is thrown.
+
+It accepts `arguments` and `environment` to start an application with them.
+
+As an usage example, `arguments` allow you to enforce language and locale for the application to start with.
+XCTest lets you start an app process by specifing [Language and Locale IDs](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html) via the process arguments with `-AppleLanguages` and `-AppleLocale`.
+Check the [Testing Specific Languages and Regions part of the Testing Your Internationalized App](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/TestingYourInternationalApp/TestingYourInternationalApp.html) for more details.
+Please terminate the application if the process was already running. The process should start with the arguments.
```

Comment:
lease terminate the application if the process was already running. The process should start with the arguments. -> Make sure to terminate the application the application before launching it with arguments if it is already running